.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # Global rule:
-*                         @expressvpn-pete-m @expressvpn-tom-l @expressvpn-andre-l @kp-mariappan-ramasamy @xv-ian-c @xv-thomas-leong
+*                         @expressvpn-pete-m @expressvpn/expressvpn-lightway-codeowners
```

Comment:
GH says this is invalid. 
(BTW, Pete is in the team)

Reply:
Yes, looks like I need to make the GH team publicly visible. Working on that now!

Reply:
Updated the Github team to be visible and given write access to this repo! Automation now declares this CODEOWNERS file to be valid :)